website/pages/en/cookbook/substreams-powered-subgraphs.mdx

```diff
@@ -4,15 +4,25 @@ title: Substreams-powered subgraphs
 
 [Substreams](/substreams/README) is a new framework for processing blockchain data, developed by StreamingFast for The Graph Network. A substreams modules can output entity changes, which are compatible with Subgraph entities. A subgraph can use such a Substreams module as a data source, bringing the indexing speed and additional data of Substreams to subgraph developers.
 
-> This cookbook uses this [Substreams-powered subgraph as a reference](https://github.com/graphprotocol/graph-tooling/tree/main/examples/substreams-powered-subgraph).
-
 ## Requirements
 
-This cookbook requires [yarn](https://yarnpkg.com/) as well as [the dependencies necessary for local Substreams development](https://substreams.streamingfast.io/developers-guide/installation-requirements).
+This cookbook requires [yarn](https://yarnpkg.com/), [the dependencies necessary for local Substreams development](https://substreams.streamingfast.io/developers-guide/installation-requirements), and the latest version of Graph CLI (>=0.52.0):
+
+```
+npm install -g @graphprotocol/graph-cli
```

Comment:
I find it funny that we say it requires `yarn` but then go on and use `npm`. I guess it doesn't really matter for global packages, and anyone who has Node installed will also have `npm`.